web/cypress/integration/history/history.spec.js

```diff
@@ -1,8 +1,4 @@
 context('TASKANA History', () => {
   beforeEach(() => cy.loginAs('admin'));
 
-  it('should display the history', () => {
-    cy.visit(Cypress.env('appUrl') + '/history');
-    cy.get('.table > form > .table-body > .table-row').should('have.length.greaterThan', 10);
-  });
```

Comment:
Why did you remove this test?

Reply:
I have deleted this test as the history plugin is started optionally. But you are right, I should add it again and set a corresponding property in the config files which is checked within the test.